Source/WebCore/Modules/cookie-store/CookieListItem.h

```diff
@@ -36,15 +36,35 @@ namespace WebCore {
 struct CookieListItem {
     CookieListItem() = default;
 
-    CookieListItem(Cookie&& cookie)
+    CookieListItem(const Cookie& cookie)
```

Comment:
Please double check that this new constructor is still needed after the changes I am suggesting below.

Reply:
It's not needed anymore so I have removed and not inlined the body of the constructor to a new function